bedrock/base/templates/macros-protocol.html

```diff
@@ -250,6 +250,7 @@
         alt=image_alt,
         class='mzp-c-card-image',
         height=image_height,
+        highres_image_url=highres_image_url,
         include_highres=include_highres_image,
```

Comment:
"include_highres" should already provide the highres image but there seems to be a disconnect between our www-admin data and the context data

[www-admin](https://github.com/mozmeao/www-admin/blob/master/content/home-de/card_1/de.md) says include_highres should be true
<img width="1062" alt="www-admin" src="https://user-images.githubusercontent.com/19650432/193618904-0d9131ae-3202-4666-9d8c-6f8d764279af.png">


context data (printed from [view file](https://github.com/mozilla/bedrock/blob/main/bedrock/mozorg/views.py#L160)) doesn't have an include_highres_image param
<img width="697" alt="context data" src="https://user-images.githubusercontent.com/19650432/193617272-4c5bf73f-862c-42bf-98f7-0a4327db0871.png">

card macro uses default include_highres_image False (image from logging include_highres and image_url)
<img width="752" alt="macro" src="https://user-images.githubusercontent.com/19650432/193617810-a2cd2a84-a374-40e0-8c63-14ed055b940c.png">

Reply:
After investigating, the missing piece between www-admin and context data is in `process_data`: https://github.com/mozmeao/www-admin/blob/master/process_data.py#L155

Given the way the image hashing works, the bedrock helper would output `home-de/card_1/vpn.e631cc1760f3-high-res.jpg` instead of `home-de/card_1/vpn-high-res.e631cc1760f3.jpg` and that would 404. Getting the high res image url from `process_card_images` instead of the `high_res_img` macro (current behaviour) is best.